bigquery_etl/cli/query.py

```diff
@@ -1349,94 +1355,113 @@ def initialize(ctx, name, sql_dir, project_id, dry_run, parallelism):
         )
         sys.exit(1)
 
-    for query_file in query_files:
+    def _initialize(query_file):
+        client = bigquery.Client()
+        project, dataset, destination_table = extract_from_query_path(query_file)
```

Comment:
For the `init.sql` approach I think it was important for the client to be initialized with target project, because the table/view names in the `CREATE TABLE` and `CREATE MATERIALIZED VIEW` statements might not be fully qualified.
```suggestion
        project, dataset, destination_table = extract_from_query_path(query_file)
        client = bigquery.Client(project=project)
```
(this was previously done by instantiating a new client, but your latest revision removes that code)